src/webview/MessageList.js

```diff
@@ -72,6 +72,7 @@ export type BackgroundData = $ReadOnly<{
   flags: FlagsState,
   mute: MuteState,
   ownUser: User,
+  theme: ThemeName,
```

Comment:
I'm not sure what jsdoc you're referring to when you say, in the commit message,

> as explained in the jsdoc on the type.

The jsdoc on `BackgroundData` in particular states: `Data about the user, the realm, and all known messages.`, which doesn't seem consonant with including `theme`.

Reply:
The theme is a global choice made by the user, so I think it fits broadly under "about the user".

The main reason I think it belongs here is the other side of the distinction, in the next paragraph: it's independent of the specific narrow and specific messages we're displaying.

Which also goes along with being suitable to include in the bundle that's wired through to a diverse set of helpers below: many of those helpers deal with a particular message, and when they also get data about the current narrow, that creates a risk of confusion (just like in #3811) about which information applies.